syllabus.md

```diff
@@ -63,13 +71,33 @@ Students who are interested in knowing how to map percentage grades onto letter
+- Copying from other students homework, websites, old solutions, etc. is cheating and offenders will be reffered to the Center for Student Conduct.
+
 ## Late Policy 
 
 We set deadlines for homework, tests, and labs to be completed. These deadlines are to your advantage, becacuse they help you to prioritize your learning. At the same time, we understand that MIDS a single facet of the life that you live. 
 
-- **Homework**: Homework assignments are chances for you to apply the content that you have learned in the async and the live sessions. These occur at a regular (nearly weekly) pace. Homework is due _at the time that your live session meets_ on the day that it meets. 
-  - Students have five "late days" that they can use without penalty on homework through the semester. After those "late days" have been used, each day late will be assessed a 10% penalty from the final grade of that homework assignment. As an example, after using all five late days, a homework assignment turned in one day late could earn a maximum score of a 90% _on that homework_. 
-  - Students cannot turn in any single homework more than two days late. This balances the need for student flexibility, with the need to release feedback to other students in a section. For example, a student who has class on Monday at 4:00p cannot turn in an assignment later than Wednesday at 4:00p.
+- **Homework**: Homework assignments are chances for you to apply the content that you have learned in the async and the live sessions. These occur at a regular (nearly weekly) pace. **Homework is due _at the time that your live session meets_ on the day that it meets.** 
+  - Students have **five "late days" that they can use without penalty** on homework through the semester. After those "late days" have been used, each day late will be assessed a 10% penalty from the final grade of that homework assignment. As an example, after using all five late days, a homework assignment turned in one day late could earn a maximum score of a 90% _on that homework_. 
+  - Students **cannot turn in any single homework more than two days late.** This balances the need for student flexibility, with the need to release feedback to other students in a section. For example, a student who has class on Monday at 4:00p cannot turn in an assignment later than Wednesday at 4:00p.
+  - _Note on Gradescope due dates:_ Gradescope is the online platform that we use for homework. The due dates shown on gradescope are there to accomodate the last section of the week. You still must turn in your assignments before your live session meets. Lateness will be evaluated at the end of the quarter. When final grades are tallied. You must track how many late days you have used
```

Comment:
I think we technically have trimesters?  I feel like most folks in the program use semester though.  
```suggestion
  - _Note on Gradescope due dates:_ Gradescope is the online platform that we use for homework. The due dates shown on gradescope are there to accomodate the last section of the week. You still must turn in your assignments before your live session meets. Late days will be counted at the end of the semester.  You must keep your own tally of how many late days you have used.
```